services/workshop/crapi/shop/tests.py

```diff
@@ -163,3 +166,35 @@ def test_sql_injection(self):
             self.user.number + ' ' + messages.COUPON_ALREADY_APPLIED
         )
 
+    def Test_create_order(self):
+        """
+        creates an order using the product_id from the previous test.
+        should get a valid response saying product created
+        :return: order details 
+        """
+        order_details = {
+            "product_id":  str(self.product_id),
+            "quantity": 1
+          
+        }
+        res = self.client.post('/workshop/api/shop/orders', order_details, content_type='application/json', **self.auth_headers)
+        self.order_id = json.loads(res.content)['id']
+        self.assertEqual(res.status_code, 200)
+
+    def Test_unauthenticated_get_order(self):
```

Comment:
Remove Test_ from the non test case. Helper functions should not start with Test_

Reply:
Noted, I have removed them.